### PR TITLE
Add a new RPC to rename application configuration file

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/server/grpcapi/api.go [hunk 1]
```diff
@@ -310,6 +310,29 @@ func (a *API) ListApplications(ctx context.Context, req *apiservice.ListApplicat
...
+func (a *API) RenameApplicationConfigFile(ctx context.Context, req *apiservice.RenameApplicationConfigFileRequest) (*apiservice.RenameApplicationConfigFileResponse, error) {
+	key, err := requireAPIKey(ctx, model.APIKey_READ_WRITE, a.logger)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, appID := range req.ApplicationIds {
+		err := a.applicationStore.UpdateApplication(ctx, appID, func(app *model.Application) error {
+			if app.ProjectId != key.ProjectId {
+				return fmt.Errorf("Requested application %s does not belong to your project", appID)
```
Comment (nakabonne) [inline pkg/app/server/grpcapi/api.go:322]: Honestly, in this case, we would like to return BadRequest or something like that. But it's on you to overdo it.

Comment (nghialv) [reply pkg/app/server/grpcapi/api.go:322]: You are right!
My lazy part decided to ignore that thing. But let me fix it to be right.

Comment (nghialv) [reply pkg/app/server/grpcapi/api.go:324]: @nakabonne Fixed it!

**Comment by nakabonne** [comment]
Looks cooler!
/lgtm

**Comment by khanhtc1202** [comment]
Here you go 🚀 
/approve

**Pull request merged**